android/src/com/mapswithme/maps/search/HotelsFilter.java

```diff
@@ -0,0 +1,90 @@
+package com.mapswithme.maps.search;
+
+import android.support.annotation.NonNull;
+
+public class HotelsFilter
```

Comment:
Yes, we can. But I saw a lot of places in our Java code where enumerations are implemented just like a set of named constants, not as enum.